<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/avatar/default_avatar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Timekeeping Record</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Timekeeping">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>200</y>
      <width>381</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time In</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time Out</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tbl_Employee">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>551</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Full Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department ID</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tbl_Department">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>371</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="btn_Back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Search_Department">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Search_Employee">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_InTime">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>160</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Go In Time</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_GoLate">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>160</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Late</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>160</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Timekeeping">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>810</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total Record: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Department">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>810</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total Department: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Employee">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>810</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total Employee : 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="btn_Search_Department">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="btn_Search_Employee">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
